e2etest/pkg/frr/container/container.go

```diff
@@ -365,3 +374,30 @@ func containerIsRunning(containerName string) error {
 
 	return nil
 }
+
+func SetupP2PPeer(frrImage string, node corev1.Node) (*FRR, error) {
+	c := Config{
+		Name:    fmt.Sprintf("unnumbered-p2p-%d", time.Now().Unix()),
+		Image:   frrImage,
+		Network: "none",
+		Router: frrconfig.RouterConfig{
+			ASN:     650001,
+			BGPPort: 179,
+		},
+	}
+
+	peers, err := Create(map[string]Config{"peer": c})
+	if err != nil {
+		return nil, fmt.Errorf("create container failed - %w", err)
+	}
+
+	peer := peers[0]
+
+	if err := frrconfig.WirePeer(peer.Name, node); err != nil {
+		return nil, fmt.Errorf("wire the peer failed - %w", err)
+	}
+	if err := peer.UpdateBGPConfigFile(frrconfig.UnnumberedPeerFRRConfig); err != nil {
+		return nil, err
+	}
```

Comment:
imo this part does not belong as part of this function, we should keep this package around "containers setup", and call the frr bgp config parts where needed in the tests / where it makes more sense

Reply:
done, will be added in the ffr-k8s repo in the test